indico_hub/schemas.py

```diff
@@ -3,3 +3,46 @@
 
 
 # TODO: Implement Marshmallow schemas for the API request data
+
+from flask_marshmallow import *
+from marshmallow import *
+from marshmallow_sqlalchemy import *
+from webargs.core import T
+from .models import *
```

Comment:
please don't ever use `import *` - it makes code hard to understand and static analyzers like flake8 much less useful

Reply:
solved